src/scripts/main.js

```diff
@@ -1,3 +1,33 @@
 'use strict';
 
-// write your code here
+const logo = document.querySelector('.logo');
+
+const promise1 = new Promise((resolve) => {
+  logo.addEventListener('click', () => {
+    resolve();
+  });
+});
+
+const promise2 = new Promise((resolve, reject) => {
+  setTimeout(() => reject(new Error()), 3000);
+});
+
+const successHandler = () => {
+  const div = document.createElement('div');
+
+  div.classList.add('message');
+  div.textContent = 'Promise was resolved!';
+  document.body.append(div);
+};
+
+const errorHandler = () => {
+  const div = document.createElement('div');
+
+  div.classList.add('message');
+  div.classList.add('error-message');
+  div.textContent = 'Promise was rejected!';
+  document.body.append(div);
+};
+
+promise1.then(successHandler).catch(errorHandler);
+promise2.then(successHandler).catch(errorHandler);
```

Comment:
The current implementation handles `promise1` and `promise2` separately, which means both handlers will execute independently. If you want to handle them together, consider using `Promise.race` or `Promise.all` to manage their outcomes collectively. This way, you can determine which promise resolves or rejects first and handle it accordingly.